libraries/chef_artifact.rb

```diff
@@ -102,8 +102,11 @@ def retrieve_from_s3(node, source_file, destination_file)
           # Set Config
           Aws.config[:region] = region
           if config.empty?
-            Chef::Log.debug("Config not found for databag AWS. Using Instance Profile Credentials")
-            Aws.config[:credentials] = Aws::InstanceProfileCredentials.new()
+            Chef::Log.info("Config empty, supplying defaults...")
+            Aws.config[:credentials] = Aws::Credentials.new(
+              node['deploy']['release']['aws_access_key_id'],
```

Comment:
Let's not hard-code deploy cookbook attributes in this cookbook. I'd rather not set any at all and require the calling cookbook to configure.